web-server/prisma/datamodel.prisma

```diff
@@ -1,6 +1,7 @@
 type Url {
-  id: ID! @unique
-  createdAt: DateTime!
+  id: ID! @id
```

Comment:
Can you explain why this needs to change?  It would be great to have this in a comment if possible.

Reply:
v1.32 of Prisma introduced a new datamodel syntax that was a breaking change. The `id`, `createdAt`, and `updatedAt` fields now allow custom naming for the field while still maintaining the behaviors for the field type as a generator. For example, the following key and value would be valid under this schema: `myId: ID! @id` 

For more on this change: https://www.prisma.io/blog/datamodel-v11-lrzqy1f56c90#use-any-field-as-id-createdat-or-updatedat